pandas/util/_test_decorators.py

```diff
@@ -266,7 +266,8 @@ def file_leak_context():
     ContextManager analogue to check_file_leaks.
     """
     psutil = safe_import("psutil")
-    if not psutil:
+    if not psutil or is_platform_windows():
+        # Checking for file leaks can hang on Windows CI
```

Comment:
is there a reference for this claim?

Reply:
Just anecdotally while investigating https://github.com/pandas-dev/pandas/pull/45478.

I noticed disabling the analog check in the excel tests would no longer hang the build around the excel files